tests/decider_tests.py

```diff
@@ -480,16 +486,17 @@ def test_none_returned_on_variant_call_with_bad_id(self):
             }
         }
         with create_temp_config_file(config) as f:
-            decider = self.setup_decider(f.name, self.minimal_decider_context)
-
             with self.assertLogs() as captured:
+                decider = setup_decider(
+                    f, self.minimal_decider_context, self.mock_span, self.event_logger
+                )
                 variant = decider.get_variant("test")
 
                 self.assertEqual(variant, None)
                 self.assertEqual(self.event_logger.log.call_count, 0)
 
                 assert any(
-                    'Rust decider has initialization error: Decider initialization failed: Json error: "invalid type: string \\"1\\"'
+                    'rust_decider.init() has error: Decider initialization failed: Partially loaded decider: 1 features failed to load.'
```

Comment:
@gshipilov until I incorporate de-structuring the [hashmap](https://github.snooguts.net/reddit/decider/blob/7c81a56bea90a1b741b596cbed6a0eb98dc8cbf4/decider/src/lib.rs#L887) of feature specific errors, we're currently losing error info, we're losing that info for now.

Reply:
Yup, I think this is ok. I'm assuming restructuring the partial error is gonna happen relatively soon, and we have some workarounds for this if we end up in an incident.